examples/pl_domain_templates/reinforce_learn_Qnet.py

```diff
@@ -368,7 +368,7 @@ def get_device(self, batch) -> str:
 
 def main(args) -> None:
     model = DQNLightning(**vars(args))
-    trainer = Trainer(accelerator="gpu", devices=1, val_check_interval=100)
+    trainer = Trainer(accelerator="cpu", devices=1, val_check_interval=100)
```

Comment:
Should we try auto?

Reply:
I specifically chose cpu because I'm running this on a apple silicon mac, and there it runs into an unsupported operation (PT 2.0), crashing.